docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md

```diff
@@ -179,8 +179,6 @@ public override Task Run()
             Console.WriteLine();
         }
     }
-
-    return Task.CompletedTask;
```

Comment:
I see we're still returning this from `Initialize`. Is there a motivation for when or when not to return a completed task vs. nothing vs. `return base.{whatever}`?

Reply:
If you add async to the signature the compiler takes care of it. If you're not awaiting any calls, then there's no point in adding async to the signature ... but then we lose the automagical async/await compiler cleanup so we have to manually return a Task.

Reply:
That kind of makes sense, but in this case, we don't have `async` or a returned task anymore, do we? And even if we `await` something in our override, wouldn't returning/awaiting the `base` implementation be "better". (Though in this case, I am guessing they are basically null implementations.)